ls1intum/Artemis
# `Development`: Add documentation for local version control and continuous integration setup

### Checklist
#### General

### Description
This PR adds documentation for how to set up the local VC and locl CI system locally.

### Steps for Testing

- Check the documentation for the spots affected by this PR.
- If you have not done so, set the system up on your local machine using the provided documentation and make sure the steps are comprehensible.

### Review Progress
#### Code Review
#### Manual Tests
